pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -234,6 +238,24 @@ func manageClientCABundle(lister corev1listers.ConfigMapLister, client coreclien
 	return resourceapply.ApplyConfigMap(client, recorder, requiredConfigMap)
 }
 
+func manageKubeletServingCABundle(lister corev1listers.ConfigMapLister, client coreclientv1.ConfigMapsGetter, recorder events.Recorder) (*corev1.ConfigMap, bool, error) {
+	requiredConfigMap, err := resourcesynccontroller.CombineCABundleConfigMaps(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "kubelet-serving-ca"},
+		lister,
+		nil, // TODO remove this
+		nil, // TODO remove this
+		// this is from the installer and contains the value they think we should have
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalUserSpecifiedConfigNamespace, Name: "initial-kubelet-serving-ca"},
+		// this is from kube-controller-manager and indicates the ca-bundle.crt to verify the signed kubelet serving certs
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "csr-controller-ca"},
```

Comment:
who creates and rotates that?

Reply:
and what happens if the configmap does not exist? Is it just ignored?

Reply:
> and what happens if the configmap does not exist? Is it just ignored?

yes, ignored.